frontend/public/components/utils/kebab.tsx

```diff
@@ -134,6 +144,18 @@ const kebabFactory: KebabFactory = {
 // The common menu actions that most resource share
 kebabFactory.common = [kebabFactory.ModifyLabels, kebabFactory.ModifyAnnotations, kebabFactory.Edit, kebabFactory.Delete];
 
+export const getExtensionsKebabActionsForKind = (kind: K8sKind) => {
```

Comment:
I think you can simplify this to:
```suggestion
export const getExtensionsKebabActionsForKind = (kind: K8sKind) => plugins.registry.getKebabActions().filter(e => e.properties.getKebabActionsForKind(kind);
```